app/workers/email_worker.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class EmailWorker
```

Comment:
I'm not pretty sure if this running anytime. Can you check this @hdf1986 ?

Reply:
Hmm no, it's not needed. Please remove it :)